[nodejs] Add cycles 13, 8

BiNZGi opened the pull request:
From #525
Readd from https://github.com/endoflife-date/endoflife.date/commit/f3cf737e2a92d9914eaf5693530f09fef7bf3082#diff-252b7c5f491ceacafa0a6744a80913e26befedc4461b3d019af0eab91ddd2db0

BiNZGi committed:
[nodejs] Readd cycles 13, 8

From #525
Readd from https://github.com/endoflife-date/endoflife.date/commit/f3cf737e2a92d9914eaf5693530f09fef7bf3082#diff-252b7c5f491ceacafa0a6744a80913e26befedc4461b3d019af0eab91ddd2db0

BiNZGi committed:
[nodejs] Update 13.14.0, 8.16.2 from release-data

https://github.com/endoflife-date/release-data/blob/main/releases/nodejs.json

captn3m0 commented on products/nodejs.md:
```diff
@@ -63,7 +69,12 @@ releases:
-
+-   releaseCycle: "8"
+    release: 2017-10-31
+    lts: true
```
I think Web Archive has some of the older LTS start dates.

BiNZGi replied:
Thanks for the hint, found: https://web.archive.org/web/20190317014458/https://nodejs.org/en/about/releases/

BiNZGi committed:
[nodejs] Update 8 lts date

https://web.archive.org/web/20190317014458/https://nodejs.org/en/about/releases/

BiNZGi committed:
[nodejs] Fix releaseDate
